node-registrar/client/mnemonic.go

```diff
@@ -0,0 +1,44 @@
+package client
+
+import (
+	"github.com/cosmos/go-bip39"
+	"github.com/pkg/errors"
+	subkeyEd25519 "github.com/vedhavyas/go-subkey/v2/ed25519"
+
+	"github.com/vedhavyas/go-subkey/v2"
+)
+
+func (c *RegistrarClient) Mnemonic() string {
+	return c.mnemonic
+}
+
+func parseKeysFromMnemonicOrSeed(mnemonicOrSeed string) (keypair subkey.KeyPair, err error) {
+	// otherwise drive key pair from seed
+	keypair, err = subkey.DeriveKeyPair(subkeyEd25519.Scheme{}, mnemonicOrSeed)
```

Comment:
won't sr25519 be supported?

Reply:
it's supported in the server, if needed we can support them too in the client, we need to pick a default tho